src/partitioner/local/MetisPartitioner.cpp

```diff
@@ -280,6 +287,50 @@ void MetisPartitioner::createPartitionFiles(idx_t *part) {
         std::map<int, std::vector<int>> partEdgeMap = partitionedLocalGraphStorageMap[part];
         std::map<int, std::vector<int>> partMasterEdgeMap = masterGraphStorageMap[part];
 
+        std::map<long, string[7]> partitionedEdgeAttributes;
```

Comment:
What is this 7? Can't you have a constant declared?

Reply:
It is an array of size 7.

Reply:
Can you declare this 7 as a constant and refer it?

Reply:
I tried to do it but it prompts an error.